src/components/vue-form-renderer.vue

```diff
@@ -299,6 +300,11 @@ export default {
         defaultValue = [];
       }
 
+      if (item.component === 'FormDatePicker') {
+        let date = new Date();
```

Comment:
Since you're not re-assigning `date`, you can use `const` instead of `let`.

Reply:
New commits include removing the `defaultValues` data key and updating `let` to `const` for the `date` variable. 😃